quipucords/api/messages.py

```diff
@@ -11,10 +11,18 @@
 
 # host credential messages
 PLURAL_HOST_CREDENTIALS_MSG = "Credentials"
-HC_PWD_OR_KEYFILE = "A host credential must have either a password or an ssh_keyfile."
-HC_NOT_BOTH = (
+HC_KEYFILE_OR_KEYVALUE = (
+    "A host credential must have either an ssh_keyfile or an ssh_keyvalue, not both."
+)
+HC_PWD_OR_KEYFILE_OR_KEYVALUE = (
+    "A host credential must have either a password, ssh_keyfile or an ssh_keyvalue."
```

Comment:
```suggestion
    "A host credential must have a password, ssh_keyfile, or ssh_keyvalue exclusively."
```
Swapping "either" since that should be used only for a list of two items, and adding the oxford comma since our docs friends agreed to its use elsewhere.